controllers/pins-controller.js

```diff
@@ -0,0 +1,86 @@
+const {
+  selectAllPins,
+  selectPin,
+  addPin,
+  modifyPin,
+  removePin,
+} = require('../models/pins-models');
+
+function getAllPins(req, res, next) {
+  const acceptQueries = [
+    'sort_by',
+    'order',
+    'creator', // users.name
```

Comment:
Are we using `creator` then, or `username`?

Reply:
Changed to username, username uniqueness enforced